VocaDbModel/Service/QueryableExtensions/CommentQueryableExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Linq;
+using VocaDb.Model.Domain;
+using VocaDb.Model.Domain.Comments;
+
+namespace VocaDb.Model.Service.QueryableExtensions
+{
+	public static class CommentQueryableExtensions

```

Comment:
Could you just make Comment implement IDeletableEntry?

Reply:
Originally I intended the comment ID to be int64, so I didn't make `Comment` implement `IDeletableEntry`.